lib/fluent/command/fluentd.rb

```diff
@@ -155,7 +155,7 @@
   cmd_opts[:strict_config_value] = b
 }
 
-op.on('--enable-input-metrics', "Enable input plugin metrics on fluentd", TrueClass) {|b|
+op.on('--enable-input-metrics', "[DEPRECATED] Enable input plugin metrics on fluentd", TrueClass) {|b|
```

Comment:
Thanks!

It would be possible for us to leave this option as is and only change the default value.
Thus, if this affects performance, a user can disable it.
What do you think about this?

Reply:
Ah, that's good idea.
Turn this option by default and leave the choice of disabling this should be better than completely removed it.

Reply:
I added an option to disable it at startup.

Reply:
If possible, I want to use the existing option because it will allow us to avoid making a new deprecated option.

Could we do either of the following?

```rb
--[no-]enable-input-metrics
```
=> a user would be able to disable this by `--no-enable-input-metrics`

or

```rb
--enable-input-metrics [BOOL] ... do |b|
  opt = !!b
end
```
=> a user would be able to disable this by `--enable-input-metrics no`

Reply:
I'm not sure if either of these are common ways.

Reply:
I see!

For users, it would be clearer if the feature could be disabled with the option `--disable-input-metrics`.

So, looks good with the current specification. Thanks!

Reply:
Oh, sorry, I have one question.
What about systemconfig?

We have `--disable-input-metrics` command line option.
But we still use `enable_input_metrics` in systemconfig?

Reply:
The `--enable-input-metrics` option cannot be set `enable_input_metrics` to false, so I introduced the new `--disable-input-metrics` option.

However, we can set `enable_input_metrics` to false directly with system config.
I think it is sufficient to change the default values about system config.